LayoutTests/TestExpectations

```diff
@@ -6044,3 +6044,8 @@ js/ShadowRealm-iframe-detach.html [ Pass Failure ]
 js/ShadowRealm-iframe-sandboxed.html [ Pass Failure ]
 js/ShadowRealm-importValue.html [ Pass Failure ]
 js/ShadowRealm-worker.html [ Pass Failure ]
+
+# Old SVG+currentcolor tests are incorrect https://bugs.webkit.org/show_bug.cgi?id=245400
+imported/w3c/web-platform-tests/svg/import/color-prop-05-t-manual.svg [ Skip ]
+svg/W3C-SVG-1.1-SE/color-prop-05-t.svg [ Skip ]
```

Comment:
We should look into de-duplicating these tests, and fixing it upstream as well. I can take care of that.

Reply:
Yes indeed, there is bugzilla about those de-duplication tasks https://bugs.webkit.org/show_bug.cgi?id=215135